_blog/kubernetes-on-solar-plants/index.md

```diff
@@ -191,72 +179,64 @@ When a device fails, Kubernetes will reschedule all of the applications deployed
+Kubernetes is excellent for deploying containers in a way that maximises the efficiency of your infrastructure.
 
 When you deploy three instances of an application, those are scheduled to maximise efficiency.
 
-Kubernetes by default will try to spread the containers across all nodes, and try to pack your nodes as dense as possible.
+Kubernetes by default will try to spread the containers across all nodes, and try to pack your nodes as dense as possible <!-- this statement is a little contradictory, maybe delete this line? -->.
 
 There's no guarantee that three instances of your applications will end up on five different devices.
 
 {% include_relative allocations.html %}
 
-They could be deployed all on the same node.
-
-Or maybe in two, depending on the resource allocation.
+They could all be deployed on the same node, or they could be deployed across two nodes, depending on the physical resources available.
 
-Particularly in the embedded world where resources are scares, you don't want deployments just to be placed anywhere.
+Particularly in the embedded world where resources are scarce, you don't want deployments to be placed just anywhere.
 
 **You want to have a strict set of rules for deployments.**
 
 As an example, each solar panel should have only one app running at any given time.
 
-In the case of an application in charge of tilting the solar panels to track the trajectory of the sun, you don't want to have two applications deployed on the same node to drive the same motor.
+In the case of an application responsible for tilting the solar panels to track the trajectory of the sun, you don't want to have two applications deployed on the same node that are trying to drive the same motor.
 
 ## Advanced deployments in Kubernetes
 
-Kubernetes has several strategies to deploy containers in your nodes.
+Kubernetes can use several strategies to allocate containers to your nodes.
 
-The most straightforward strategy is _[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)_.
+The most straightforward strategy is a _[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)_.
 
 In a _Deployment_, you specify the number of instances of your application and Kubernetes will find the space to allocate them.
 
-This is the most common deployment.
+This is the most common deployment type, but while it's useful for cloud deployments where we don't care which specific node is running our application (we just care that it's being run by _something_), this is less useful in the embedded world.
 
-It's a very popular strategy for releasing apps in the cloud, but not useful in the embedded world.
+Other srategies include _[StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/)_ and _[ReplicaSet](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/)_. <!-- is a StatefulSet comparable here? -->
```

Comment:
I agree with you, they are probably not in the same league. But the point I'm trying to make is that they're options. If you want to (and e.g. don't care about rolling updates) you could skip Deployments and use ReplicaSets directly. So it could technically be a viable deployment strategy.